### ISSUE TITLE
FrontEnd - Menu issue on landing page

### ISSUE THREAD

**Issue by saileshpatnala** [wintersausten/latinx_news#1]
Menu on the landing page cannot be closed. On other pages, (once you've selected a country) menu button is moved and can be clicked again to close the menu.

**Comment by joeka36** [comment]
What browser were you using, and what version? It works fine on mine.

**Comment by saileshpatnala** [comment]
I'm running it on the latest version of chrome. Wonder if it is a mac issue. Let me try again

**Comment by saileshpatnala** [comment]
Made a fresh local branch and ran the server on localhost:4200 and still getting same issue
![image](https://user-images.githubusercontent.com/11470101/29648593-aed21346-8854-11e7-8ac4-dfea8f68f798.png)
Lemme know if I'm doing something wrong or whether this is a bug. Can someone else run it on a mac and check? Also ran checked safari and getting same issue.
